.github/workflows/release.yml

```diff
@@ -134,7 +134,8 @@ jobs:
             exit 1
```

Comment:
The code changes involve updating how the JIRA issue key is sanitized and stored in the environment variables. Instead of directly appending the sanitized key to $GITHUB_ENV, it is now written to a temporary file (temp.env) and then appended to $GITHUB_ENV using cat command. This change introduces an extra step but ensures that the key is correctly stored in the environment variables.